Source/sourcekitten/CompleteCommand.swift

```diff
@@ -38,7 +38,8 @@ struct CompleteCommand: CommandType {
                 args.appendContentsOf(options.compilerargs.componentsSeparatedByString(" "))
             }
             if args.indexOf("-sdk") == nil {
-                args.append("-sdk \(sdkPath())")
+                args.append("-sdk")
+                args.append("\(sdkPath())")
```

Comment:
This could just be `args.append(sdkPath())` since the function returns a `String` already, and does not need to be interpolated.